modules/apps/frontend-js/frontend-js-dropdown-support-web/src/main/resources/META-INF/resources/DropdownProvider.js

```diff
@@ -145,6 +161,12 @@ class DropdownProvider {
 			}
 		}
 	};
+
+	_maybeClickOutside = (event, {menu, trigger}) => {
+		if (isClickOutside(event.target) && event.target !== trigger) {
```

Comment:
You're passing no elements to `isClickOutside` so `filter` will have no elements to iterate over and method will always return true, even if you click inside the dropdown

Reply:
Yes, my bad. I should pass `isClickOutside(event.target, trigger, menu)` :)

See https://github.com/liferay-frontend/liferay-portal/pull/140/files/58a8f3e497edab262340011048c6be336a9800d3#r458350297